checkQC/handlers/error_rate_handler.py

```diff
@@ -47,22 +47,26 @@ def check_qc(self):
             lane_nbr = int(error_dict["lane"])
             read = error_dict["read"]
             error_rate = error_dict["error_rate"]
+            is_index_read = error_dict.get("is_index_read", False)
+            read_or_index_text = "read (I)" if is_index_read else "read"
 
             if error_rate == 0 and not self.qc_config[self.ALLOW_MISSING_ERROR_RATE]:
-                yield QCErrorFatal("Error rate was found to be 0 on lane: {} for read: {}, this is probably "
+                yield QCErrorFatal("Error rate was found to be 0 on lane: {} for {}: {}, this is probably "
                                    "because there was no PhiX loaded on this lane. If do not use PhiX for your "
                                    "runs you can set 'allow_missing_error_rate' in the config to True, which will "
-                                   "remove the messages in the future.".format(lane_nbr, read))
+                                   "remove the messages in the future.".format(lane_nbr, read_or_index_text, read))
             elif self.error() != self.UNKNOWN and error_rate > self.error():
-                yield QCErrorFatal("Error rate {} was to high on lane: {} for read: {}".format(error_rate,
+                yield QCErrorFatal("Error rate {} was to high on lane: {} for {}: {}".format(error_rate,
```

Comment:
This part would be more readable if it used f-strings and if the lines were formatted to <100 characters